dashboard/utils.py

```diff
@@ -277,6 +282,40 @@ def count_courses_passed(self):
                 passed_courses.add(self.edx_key_course_map[course_id])
         return len(passed_courses)
 
+    def get_pearson_exam_status(self):
+        """
+        Get the pearson exam status for the user / program combo
+        """
+        if self.program.exam_series_code is None:
+            return ""
+        else:
+            user = self.user
+            exam_profile = ExamProfile.objects.filter(profile=user.profile)
+
+            if not exam_profile.exists():
+                return ExamProfile.PROFILE_ABSENT
+
+            elif (exam_profile[0].status == ExamProfile.PROFILE_PENDING or
+                  exam_profile[0].status == ExamProfile.PROFILE_IN_PROGRESS):
+                return ExamProfile.PROFILE_IN_PROGRESS
+
+            elif exam_profile[0].status == ExamProfile.PROFILE_INVALID:
+                return ExamProfile.PROFILE_INVALID
+
+            elif exam_profile[0].status == ExamProfile.PROFILE_SUCCESS:
```

Comment:
I would probably put in an `else` clause here to be exhaustive. You could log an error in there and default to something sensible (maybe `PROFILE_INVALID`?). Just in case some other value gets in there and this code doesn't get updated.